examples/list_offsets.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+#
+# Copyright 2020 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# Show committed offsets and current lag for group and topic(s).
+#
+
+
+import sys
+import confluent_kafka
+
+
+if len(sys.argv) < 4:
```

Comment:
Why not just use argparse like the existing examples?

Reply:
Keeping it simple.
4 examples use sys.argv, 6 use argparse.